src/k8s-engine/wsl.ts

```diff
@@ -1390,11 +1438,14 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     try {
       this.setState(K8s.State.STOPPING);
 
-      this.lastCommandComment = 'Stopping Kubernetes';
+      this.lastCommandComment = 'Shutting Down...';
       await this.progressTracker.action(this.lastCommandComment, 10, async() => {
         if (await this.isDistroRegistered({ runningOnly: true })) {
-          await this.execCommand('/usr/local/bin/wsl-service', 'k3s', 'stop');
+          if (this.#enabledK3s) {
+            await this.execCommand('/usr/local/bin/wsl-service', '--ifstarted', 'k3s', 'stop');
```

Comment:
This doesn't need to be guarded by `this.#enabledK3s`.

Reply:
true

Reply:
`wsl-service --ifstarted k3s stop` is still being guarded by `#enableK3s`…